test/rbi/parser_test.rb

```diff
@@ -75,17 +75,26 @@ def m2; end
     def test_parse_constants
       rbi = <<~RBI
         Foo = 42
-        Bar = "foo"
+        Bar = T.let("foo", String)
         Baz = Bar
         A = nil
         B = :s
-        C = T.nilable(String)
-        D = A::B::C
+        C = T.let(nil, T.nilable(String))
+        D = T.let(A::B::C, T.class_of(A::B::C))
         A::B::C = Foo
       RBI
 
       tree = parse_rbi(rbi)
-      assert_equal(rbi, tree.string)
+      assert_equal(<<~RBI, tree.string)
+        Foo = T.let(T.unsafe(nil), T.untyped)
```

Comment:
Do we consider RBI files as documentation? Could this be a confusing change?

Reply:
Also, will this create huge RBI changes when it's adopted by Tapioca?